lib/core-client/src/preview/start.ts

```diff
@@ -39,6 +40,7 @@ export function start<TFramework extends AnyFramework>(
     globalWindow.IS_STORYBOOK = true;
   }
 
+  // TODO: talk to tom about the removal of this
```

Comment:
@tmeasday let's chat about this, how to remove.

Reply:
We are good to remove this also. We will need to rewrite all the renderers to not call this any more in v6 mode. Again, I have a refactor in mind here, best to wait for that.